full_stack_projects/aa_approved_full_stack_projects.md

```diff
@@ -0,0 +1,222 @@
+- Feature 2: Search/Filters 
+- Feature 3: Reviews/Ratings - _CRUD_
+- Feature 4: Map - _Google Maps_
+- Bonus: Mark reviews funny, cool, useful etc.
+- Bonus: Profile
+- Bonus: Friends
+
+## Open Table 
+- Feature 1: Restaurants
+- Feature 2: Reservations - _CRUD_
+- Feature 3: Search
+- Feature 4: Ratings/Reviews - _CRUD_
+- Bonus: Favorites
+- Bonus: Discover/Explore
+- Bonus: points for bookings and usage of site
+
+## Streeteasy 
+- Feature 1: Listings - _CRUD_
+- Feature 2: Favorites - _CRUD_
+- Feature 3: Map - _Google Maps_
+- Feature 4: Search
+- Bonus: Ask a question
+- Bonus: Blog
+
+## Zillow
+- Feature 1: Listings - _CRUD_
+- Feature 2: Favorites - _CRUD_
+- Feature 3: Map - _Google Maps_
+- Feature 4: Search
+- Bonus: Ask a question
+- Bonus: Messages
+
+## 500px
+- Feature 1: Post Pictures - _CRUD_
+- Feature 2: Follows - _CRUD_
+- Feature 3: Home Feed
+- Feature 4: Profile Page - _CRUD_
+- Bonus: Tags
+- Bonus: Discover
+- Bonus: Notifications
+
+## Etsy
+- Feature 1: Product Listings
+- Feature 2: Shopping Cart - _CRUD_
+- Feature 3: Comments/Reviews - _CRUD_
+- Feature 4: Search
+- Bonus: Categories
+- Bonus: Favorites
+
+## Amazon
+- Feature 1: Product Listings
+- Feature 2: Shopping Cart - _CRUD_
+- Feature 3: Reviews/Ratings - _CRUD_
+- Feature 4: Search
+- Bonus: Categories
+- Bonus: Lists
+
+## Eventbrite
+- Feature 1: Events - _CRUD_
+- Feature 2: Registration/Tickets - _CRUD_
+- Feature 3: Categores
+- Feature 4: Bookmark events - _CRUD_
+- Bonus: Google Maps integration
+- Bonus: Search
+
+## Flickr
+- Feature 1: Photos - _CRUD_
+- Feature 2: Albums - _CRUD_
+- Feature 3: Comments - _CRUD_
+- Feature 4: Tags - _CRUD_
+- Bonus: Favorites
+- Bonus: Follows
+
+## Goodreads
+- Feature 1: Books - _CRUD_
+- Feature 2: Reviews - _CRUD_
+- Feature 3: Bookshelves - _CRUD_
+- Feature 4: Search - _CRUD_
+- Bonus: Review Likes
+- Bonus: Friends
+
+## Medium
+- Feature 1: Stories - _CRUD_
+- Feature 2: Commenting on stories - _CRUD_
+- Feature 3: Follows / Feed - _CRUD_ 
+- Feature 4: Likes - _CRUD_
+- Bonus: Topics/categories
+- Bonus: Bookmarks
+
+## LinkedIn
+- Feature 1: Profile - _CRUD_
+- Feature 2: Connections - _CRUD_
+- Feature 3: Posts - _CRUD_
+- Feature 4: Feed
+- Bonus: Notifications
+- Bonus: Messaging
+
+## Pinterest
+- Feature 1: Profile - _CRUD_
+- Feature 2: Boards and Pins - _CRUD_
+- Feature 3: Follows - _CRUD_
+- Feature 4: Discover Feed
+- Bonus: Notifications
+- Bonus: Private boards
+- Bonus: Likes
+
+## Shopping sites
+- Feature 1: Product Listings
+- Feature 2: Shopping Cart - _CRUD_
+- Feature 3: Reviews/Comments - _CRUD_
+- Feature 4: Search
+- Bonus: Categories
+- Bonus: Favorites/Lists
+
+## Tumblr
+- Feature 1: Posts - _CRUD_
+- Feature 2: Feed
+- Feature 3: Follows - _CRUD_
+- Feature 4: Likes - _CRUD_
+- Bonus: Reblog
+- Bonus: User show page (blog)
+
+## Facebook
+- Feature 1: Profiles - _CRUD_
+- Feature 2: Friending - _CRUD_
+- Feature 3: Comments/Posting on each others' walls - _CRUD_
```

Comment:
For the sake of makign sure people's projects have content, it's typically recommended to do Posts before Friends.

Reply:
Updated in next push